ADAL/tests/integration/AADAuthorityValidationIntegrationTests.m

```diff
@@ -54,35 +54,57 @@ - (void)tearDown
 }
 
 //Does not call the server, just passes invalid authority
-- (void)testValidateAuthorityError
+
+
+
+- (void)testValidateAuthority_whenSchemeIsHttp_shouldFailWithError
```

Comment:
These aren't unit tests, by design they do multiple things and the unit testing conventions do not hold.